packages/ember-data/lib/system/relationships/state/belongs_to.js

```diff
@@ -16,6 +16,21 @@ BelongsToRelationship.prototype = Ember.create(Relationship.prototype);
 BelongsToRelationship.prototype.constructor = BelongsToRelationship;
 BelongsToRelationship.prototype._super$constructor = Relationship;
 
+BelongsToRelationship.prototype.updateFromAdapter = function(data) {
+  var key   = this.key,
+      value = data[this.key];
```

Comment:
Can you put another var here please? It makes debugging easier to have multiline var assignments